editing oauth connector fields fails silently

To reproduce with Salesforce on cloud:
1. Create a source with Salesforce with OAuth
2. Try to edit the start date.
3. Get this error in the console:
```
instrument.ts:129 TypeError: Cannot read properties of undefined (reading 'value')
    at children (MainInfo.tsx:141)
    at createFormattedComponent.js:46
    at Vs (react-dom.production.min.js:274)
    at ju (react-dom.production.min.js:250)
    at Su (react-dom.production.min.js:250)
    at Eu (react-dom.production.min.js:250)
    at mu (react-dom.production.min.js:243)
    at react-dom.production.min.js:123
    at t.unstable_runWithPriority (scheduler.production.min.js:18)
    at qi (react-dom.production.min.js:122)
    at $i (react-dom.production.min.js:123)
    at Vi (react-dom.production.min.js:122)
    at fu (react-dom.production.min.js:237)
    at To (react-dom.production.min.js:170)
    at SourceSettings.tsx:67
    at c (runtime.js:63)
    at Generator._invoke (runtime.js:293)
    at Generator.throw (runtime.js:118)
    at r (asyncToGenerator.js:3)
    at s (asyncToGenerator.js:29)
```
4. Get instantly redirected to the source page.
5. Go back to the source settings and see the old value (due to the error).

I'm also getting a 500 error when clicking on the "Authenticate your Salesforce account" button in the edit page. Not sure if that's related or not. 

I think there are up to three things to fix here:
1. Editing fields for OAuth connectors
2. Reauthenticating Oauth on the source page
3. Making sure any errors like this TypeError are caught and shown as errors when submitting the form instead of silently failing in the future.

Error I'm seeing:
```
{"message":"The provided configuration does not fulfill the specification. Errors: json schema validation failed when comparing the data to the json schema. \nErrors: $.client_id: is missing but it is required, $.client_secret: is missing but it is required \nSchema: \n{\n  \"type\" : \"object\",\n  \"title\" : \"Salesforce Source Spec\",\n  \"$schema\" : \"http://json-schema.org/draft-07/schema#\",\n  \"required\" : [ \"client_id\", \"client_secret\", \"refresh_token\", \"start_date\", \"api_type\" ],\n  \"properties\" : {\n    \"api_type\" : {\n      \"enum\" : [ \"BULK\", \"REST\" ],\n      \"type\" : \"string\",\n      \"default\" : \"BULK\",\n      \"description\" : \"Unless you know that you are transferring a very small amount of data, prefer using the BULK API. This will help avoid using up all of your API call quota with Salesforce. Valid values are BULK or REST.\"\n    },\n    \"client_id\" : {\n      \"type\" : \"string\",\n      \"description\" : \"The Consumer Key that can be found when viewing your app in Salesforce\"\n    },\n    \"is_sandbox\" : {\n      \"type\" : \"boolean\",\n      \"default\" : false,\n      \"description\" : \"Whether or not the the app is in a Salesforce sandbox. If you do not know what this, assume it is false. We provide more info on this field in the <a href=\\\"https://docs.airbyte.io/integrations/destinations/salesforce#is_sandbox\\\">docs</a>.\"\n    },\n    \"start_date\" : {\n      \"type\" : \"string\",\n      \"pattern\" : \"^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}Z$\",\n      \"examples\" : [ \"2021-07-25T00:00:00Z\" ],\n      \"description\" : \"UTC date and time in the format 2017-01-25T00:00:00Z. Any data before this date will not be replicated. This field uses the \\\"updated\\\" field if available, otherwise the \\\"created\\\" fields if they are available for a stream.\"\n    },\n    \"client_secret\" : {\n      \"type\" : \"string\",\n      \"description\" : \"The Consumer Secret that can be found when viewing your app in Salesforce\",\n      \"airbyte_secret\" : true\n    },\n    \"refresh_token\" : {\n      \"type\" : \"string\",\n      \"description\" : \"Salesforce Refresh Token used for Airbyte to access your Salesforce account. If you don't know what this is, follow this <a href=\\\"https://medium.com/@bpmmendis94/obtain-access-refresh-tokens-from-salesforce-rest-api-a324fe4ccd9b\\\">guide</a> to retrieve it.\",\n      \"airbyte_secret\" : true\n    }\n  },\n  \"additionalProperties\" : false\n}","exceptionClassName":"io.airbyte.server.errors.BadObjectSchemaKnownException","exceptionStack":["io.airbyte.server.errors.BadObjectSchemaKnownException: The provided configuration does not fulfill the specification. Errors: json schema validation failed when comparing the data to the json schema. ","Errors: $.client_id: is missing but it is required, $.client_secret: is missing but it is required ","Schema: ","{","  \"type\" : \"object\",","  \"title\" : \"Salesforce Source Spec\",","  \"$schema\" : \"http://json-schema.org/draft-07/schema#\",","  \"required\" : [ \"client_id\", \"client_secret\", \"refresh_token\", \"start_date\", \"api_type\" ],","  \"properties\" : {","    \"api_type\" : {","      \"enum\" : [ \"BULK\", \"REST\" ],","      \"type\" : \"string\",","      \"default\" : \"BULK\",","      \"description\" : \"Unless you know that you are transferring a very small amount of data, prefer using the BULK API. This will help avoid using up all of your API call quota with Salesforce. Valid values are BULK or REST.\"","    },","    \"client_id\" : {","      \"type\" : \"string\",","      \"description\" : \"The Consumer Key that can be found when viewing your app in Salesforce\"","    },","    \"is_sandbox\" : {","      \"type\" : \"boolean\",","      \"default\" : false,","      \"description\" : \"Whether or not the the app is in a Salesforce sandbox. If you do not know what this, assume it is false. We provide more info on this field in the <a href=\\\"https://docs.airbyte.io/integrations/destinations/salesforce#is_sandbox\\\">docs</a>.\"","    },","    \"start_date\" : {","      \"type\" : \"string\",","      \"pattern\" : \"^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}Z$\",","      \"examples\" : [ \"2021-07-25T00:00:00Z\" ],","      \"description\" : \"UTC date and time in the format 2017-01-25T00:00:00Z. Any data before this date will not be replicated. This field uses the \\\"updated\\\" field if available, otherwise the \\\"created\\\" fields if they are available for a stream.\"","    },","    \"client_secret\" : {","      \"type\" : \"string\",","      \"description\" : \"The Consumer Secret that can be found when viewing your app in Salesforce\",","      \"airbyte_secret\" : true","    },","    \"refresh_token\" : {","      \"type\" : \"string\",","      \"description\" : \"Salesforce Refresh Token used for Airbyte to access your Salesforce account. If you don't know what this is, follow this <a href=\\\"https://medium.com/@bpmmendis94/obtain-access-refresh-tokens-from-salesforce-rest-api-a324fe4ccd9b\\\">guide</a> to retrieve it.\",","      \"airbyte_secret\" : true","    }","  },","  \"additionalProperties\" : false","}","\tat io.airbyte.cloud.auth.AuthHelper.execute(AuthHelper.java:107)","\tat io.airbyte.cloud.auth.AuthHelper.callForWorkspace(AuthHelper.java:78)","\tat io.airbyte.server.wrapped.ConfigurationApiWrapped.checkConnectionToSourceForUpdate(ConfigurationApiWrapped.java:429)","\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)","\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)","\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)","\tat java.base/java.lang.reflect.Method.invoke(Method.java:564)","\tat org.glassfish.jersey.server.model.internal.ResourceMethodInvocationHandlerFactory.lambda$static$0(ResourceMethodInvocationHandlerFactory.java:52)","\tat org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher$1.run(AbstractJavaResourceMethodDispatcher.java:124)","\tat org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher.invoke(AbstractJavaResourceMethodDispatcher.java:167)","\tat org.glassfish.jersey.server.model.internal.JavaResourceMethodDispatcherProvider$TypeOutInvoker.doDispatch(JavaResourceMethodDispatcherProvider.java:219)","\tat org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher.dispatch(AbstractJavaResourceMethodDispatcher.java:79)","\tat org.glassfish.jersey.server.model.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:469)","\tat org.glassfish.jersey.server.model.ResourceMethodInvoker.apply(ResourceMethodInvoker.java:391)","\tat org.glassfish.jersey.server.model.ResourceMethodInvoker.apply(ResourceMethodInvoker.java:80)","\tat org.glassfish.jersey.server.ServerRuntime$1.run(ServerRuntime.java:253)","\tat org.glassfish.jersey.internal.Errors$1.call(Errors.java:248)","\tat org.glassfish.jersey.internal.Errors$1.call(Errors.java:244)","\tat org.glassfish.jersey.internal.Errors.process(Errors.java:292)","\tat org.glassfish.jersey.internal.Errors.process(Errors.java:274)","\tat org.glassfish.jersey.internal.Errors.process(Errors.java:244)","\tat org.glassfish.jersey.process.internal.RequestScope.runInScope(RequestScope.java:265)","\tat org.glassfish.jersey.server.ServerRuntime.process(ServerRuntime.java:232)","\tat org.glassfish.jersey.server.ApplicationHandler.handle(ApplicationHandler.java:680)","\tat org.glassfish.jersey.servlet.WebComponent.serviceImpl(WebComponent.java:394)","\tat org.glassfish.jersey.servlet.WebComponent.service(WebComponent.java:346)","\tat org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:366)","\tat org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:319)","\tat org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:205)","\tat org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:763)","\tat org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:569)","\tat org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:233)","\tat org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1377)","\tat org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:188)","\tat org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:507)","\tat org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:186)","\tat org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1292)","\tat org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141)","\tat org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:127)","\tat org.eclipse.jetty.server.Server.handle(Server.java:501)","\tat org.eclipse.jetty.server.HttpChannel.lambda$handle$1(HttpChannel.java:383)","\tat org.eclipse.jetty.server.HttpChannel.dispatch(HttpChannel.java:556)","\tat org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:375)","\tat org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:273)","\tat org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:311)","\tat org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:105)","\tat org.eclipse.jetty.io.ChannelEndPoint$1.run(ChannelEndPoint.java:104)","\tat org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.runTask(EatWhatYouKill.java:336)","\tat org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:313)","\tat org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:171)","\tat org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:129)","\tat org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:375)","\tat org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:806)","\tat org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:938)","\tat java.base/java.lang.Thread.run(Thread.java:832)","Caused by: io.airbyte.validation.json.JsonValidationException: json schema validation failed when comparing the data to the json schema. ","Errors: $.client_id: is missing but it is required, $.client_secret: is missing but it is required ","Schema: ","{","  \"type\" : \"object\",","  \"title\" : \"Salesforce Source Spec\",","  \"$schema\" : \"http://json-schema.org/draft-07/schema#\",","  \"required\" : [ \"client_id\", \"client_secret\", \"refresh_token\", \"start_date\", \"api_type\" ],","  \"properties\" : {","    \"api_type\" : {","      \"enum\" : [ \"BULK\", \"REST\" ],","      \"type\" : \"string\",","      \"default\" : \"BULK\",","      \"description\" : \"Unless you know that you are transferring a very small amount of data, prefer using the BULK API. This will help avoid using up all of your API call quota with Salesforce. Valid values are BULK or REST.\"","    },","    \"client_id\" : {","      \"type\" : \"string\",","      \"description\" : \"The Consumer Key that can be found when viewing your app in Salesforce\"","    },","    \"is_sandbox\" : {","      \"type\" : \"boolean\",","      \"default\" : false,","      \"description\" : \"Whether or not the the app is in a Salesforce sandbox. If you do not know what this, assume it is false. We provide more info on this field in the <a href=\\\"https://docs.airbyte.io/integrations/destinations/salesforce#is_sandbox\\\">docs</a>.\"","    },","    \"start_date\" : {","      \"type\" : \"string\",","      \"pattern\" : \"^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}Z$\",","      \"examples\" : [ \"2021-07-25T00:00:00Z\" ],","      \"description\" : \"UTC date and time in the format 2017-01-25T00:00:00Z. Any data before this date will not be replicated. This field uses the \\\"updated\\\" field if available, otherwise the \\\"created\\\" fields if they are available for a stream.\"","    },","    \"client_secret\" : {","      \"type\" : \"string\",","      \"description\" : \"The Consumer Secret that can be found when viewing your app in Salesforce\",","      \"airbyte_secret\" : true","    },","    \"refresh_token\" : {","      \"type\" : \"string\",","      \"description\" : \"Salesforce Refresh Token used for Airbyte to access your Salesforce account. If you don't know what this is, follow this <a href=\\\"https://medium.com/@bpmmendis94/obtain-access-refresh-tokens-from-salesforce-rest-api-a324fe4ccd9b\\\">guide</a> to retrieve it.\",","      \"airbyte_secret\" : true","    }","  },","  \"additionalProperties\" : false","}","\tat io.airbyte.validation.json.JsonSchemaValidator.ensure(JsonSchemaValidator.java:68)","\tat io.airbyte.server.handlers.SchedulerHandler.checkSourceConnectionFromSourceIdForUpdate(SchedulerHandler.java:172)","\tat io.airbyte.server.wrapped.ConfigurationApiWrapped.lambda$checkConnectionToSourceForUpdate$25(ConfigurationApiWrapped.java:432)","\tat io.airbyte.cloud.auth.AuthHelper.execute(AuthHelper.java:101)","\t... 54 more"],"rootCauseExceptionClassName":"java.lang.Class","rootCauseExceptionStack":["io.airbyte.validation.json.JsonValidationException: json schema validation failed when comparing the data to the json schema. ","Errors: $.client_id: is missing but it is required, $.client_secret: is missing but it is required ","Schema: ","{","  \"type\" : \"object\",","  \"title\" : \"Salesforce Source Spec\",","  \"$schema\" : \"http://json-schema.org/draft-07/schema#\",","  \"required\" : [ \"client_id\", \"client_secret\", \"refresh_token\", \"start_date\", \"api_type\" ],","  \"properties\" : {","    \"api_type\" : {","      \"enum\" : [ \"BULK\", \"REST\" ],","      \"type\" : \"string\",","      \"default\" : \"BULK\",","      \"description\" : \"Unless you know that you are transferring a very small amount of data, prefer using the BULK API. This will help avoid using up all of your API call quota with Salesforce. Valid values are BULK or REST.\"","    },","    \"client_id\" : {","      \"type\" : \"string\",","      \"description\" : \"The Consumer Key that can be found when viewing your app in Salesforce\"","    },","    \"is_sandbox\" : {","      \"type\" : \"boolean\",","      \"default\" : false,","      \"description\" : \"Whether or not the the app is in a Salesforce sandbox. If you do not know what this, assume it is false. We provide more info on this field in the <a href=\\\"https://docs.airbyte.io/integrations/destinations/salesforce#is_sandbox\\\">docs</a>.\"","    },","    \"start_date\" : {","      \"type\" : \"string\",","      \"pattern\" : \"^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}Z$\",","      \"examples\" : [ \"2021-07-25T00:00:00Z\" ],","      \"description\" : \"UTC date and time in the format 2017-01-25T00:00:00Z. Any data before this date will not be replicated. This field uses the \\\"updated\\\" field if available, otherwise the \\\"created\\\" fields if they are available for a stream.\"","    },","    \"client_secret\" : {","      \"type\" : \"string\",","      \"description\" : \"The Consumer Secret that can be found when viewing your app in Salesforce\",","      \"airbyte_secret\" : true","    },","    \"refresh_token\" : {","      \"type\" : \"string\",","      \"description\" : \"Salesforce Refresh Token used for Airbyte to access your Salesforce account. If you don't know what this is, follow this <a href=\\\"https://medium.com/@bpmmendis94/obtain-access-refresh-tokens-from-salesforce-rest-api-a324fe4ccd9b\\\">guide</a> to retrieve it.\",","      \"airbyte_secret\" : true","    }","  },","  \"additionalProperties\" : false","}","\tat io.airbyte.validation.json.JsonSchemaValidator.ensure(JsonSchemaValidator.java:68)","\tat io.airbyte.server.handlers.SchedulerHandler.checkSourceConnectionFromSourceIdForUpdate(SchedulerHandler.java:172)","\tat io.airbyte.server.wrapped.ConfigurationApiWrapped.lambda$checkConnectionToSourceForUpdate$25(ConfigurationApiWrapped.java:432)","\tat io.airbyte.cloud.auth.AuthHelper.execute(AuthHelper.java:101)","\tat io.airbyte.cloud.auth.AuthHelper.callForWorkspace(AuthHelper.java:78)","\tat io.airbyte.server.wrapped.ConfigurationApiWrapped.checkConnectionToSourceForUpdate(ConfigurationApiWrapped.java:429)","\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)","\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)","\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)","\tat java.base/java.lang.reflect.Method.invoke(Method.java:564)","\tat org.glassfish.jersey.server.model.internal.ResourceMethodInvocationHandlerFactory.lambda$static$0(ResourceMethodInvocationHandlerFactory.java:52)","\tat org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher$1.run(AbstractJavaResourceMethodDispatcher.java:124)","\tat org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher.invoke(AbstractJavaResourceMethodDispatcher.java:167)","\tat org.glassfish.jersey.server.model.internal.JavaResourceMethodDispatcherProvider$TypeOutInvoker.doDispatch(JavaResourceMethodDispatcherProvider.java:219)","\tat org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher.dispatch(AbstractJavaResourceMethodDispatcher.java:79)","\tat org.glassfish.jersey.server.model.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:469)","\tat org.glassfish.jersey.server.model.ResourceMethodInvoker.apply(ResourceMethodInvoker.java:391)","\tat org.glassfish.jersey.server.model.ResourceMethodInvoker.apply(ResourceMethodInvoker.java:80)","\tat org.glassfish.jersey.server.ServerRuntime$1.run(ServerRuntime.java:253)","\tat org.glassfish.jersey.internal.Errors$1.call(Errors.java:248)","\tat org.glassfish.jersey.internal.Errors$1.call(Errors.java:244)","\tat org.glassfish.jersey.internal.Errors.process(Errors.java:292)","\tat org.glassfish.jersey.internal.Errors.process(Errors.java:274)","\tat org.glassfish.jersey.internal.Errors.process(Errors.java:244)","\tat org.glassfish.jersey.process.internal.RequestScope.runInScope(RequestScope.java:265)","\tat org.glassfish.jersey.server.ServerRuntime.process(ServerRuntime.java:232)","\tat org.glassfish.jersey.server.ApplicationHandler.handle(ApplicationHandler.java:680)","\tat org.glassfish.jersey.servlet.WebComponent.serviceImpl(WebComponent.java:394)","\tat org.glassfish.jersey.servlet.WebComponent.service(WebComponent.java:346)","\tat org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:366)","\tat org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:319)","\tat org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:205)","\tat org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:763)","\tat org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:569)","\tat org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:233)","\tat org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1377)","\tat org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:188)","\tat org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:507)","\tat org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:186)","\tat org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1292)","\tat org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141)","\tat org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:127)","\tat org.eclipse.jetty.server.Server.handle(Server.java:501)","\tat org.eclipse.jetty.server.HttpChannel.lambda$handle$1(HttpChannel.java:383)","\tat org.eclipse.jetty.server.HttpChannel.dispatch(HttpChannel.java:556)","\tat org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:375)","\tat org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:273)","\tat org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:311)","\tat org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:105)","\tat org.eclipse.jetty.io.ChannelEndPoint$1.run(ChannelEndPoint.java:104)","\tat org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.runTask(EatWhatYouKill.java:336)","\tat org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:313)","\tat org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:171)","\tat org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:129)","\tat org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:375)","\tat org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:806)","\tat org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:938)","\tat java.base/java.lang.Thread.run(Thread.java:832)"]}
```

```
{"message":"The provided configuration does not fulfill the specification. Errors: json schema validation failed when comparing the data to the json schema. \nErrors: $.client_id: is missing but it is required, $.client_secret: is missing but it is required \nSchema: \n{\n  \"type\" : \"object\",\n  \"title\" : \"Salesforce Source Spec\",\n  \"$schema\" : \"http://json-schema.org/draft-07/schema#\",\n  \"required\" : [ \"client_id\", \"client_secret\", \"refresh_token\", \"start_date\", \"api_type\" ],\n  \"properties\" : {\n    \"api_type\" : {\n      \"enum\" : [ \"BULK\", \"REST\" ],\n      \"type\" : \"string\",\n      \"default\" : \"BULK\",\n      \"description\" : \"Unless you know that you are transferring a very small amount of data, prefer using the BULK API. This will help avoid using up all of your API call quota with Salesforce. Valid values are BULK or REST.\"\n    },\n    \"client_id\" : {\n      \"type\" : \"string\",\n      \"description\" : \"The Consumer Key that can be found when viewing your app in Salesforce\"\n    },\n    \"is_sandbox\" : {\n      \"type\" : \"boolean\",\n      \"default\" : false,\n      \"description\" : \"Whether or not the the app is in a Salesforce sandbox. If you do not know what this, assume it is false. We provide more info on this field in the <a href=\\\"https://docs.airbyte.io/integrations/destinations/salesforce#is_sandbox\\\">docs</a>.\"\n    },\n    \"start_date\" : {\n      \"type\" : \"string\",\n      \"pattern\" : \"^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}Z$\",\n      \"examples\" : [ \"2021-07-25T00:00:00Z\" ],\n      \"description\" : \"UTC date and time in the format 2017-01-25T00:00:00Z. Any data before this date will not be replicated. This field uses the \\\"updated\\\" field if available, otherwise the \\\"created\\\" fields if they are available for a stream.\"\n    },\n    \"client_secret\" : {\n      \"type\" : \"string\",\n      \"description\" : \"The Consumer Secret that can be found when viewing your app in Salesforce\",\n      \"airbyte_secret\" : true\n    },\n    \"refresh_token\" : {\n      \"type\" : \"string\",\n      \"description\" : \"Salesforce Refresh Token used for Airbyte to access your Salesforce account. If you don't know what this is, follow this <a href=\\\"https://medium.com/@bpmmendis94/obtain-access-refresh-tokens-from-salesforce-rest-api-a324fe4ccd9b\\\">guide</a> to retrieve it.\",\n      \"airbyte_secret\" : true\n    }\n  },\n  \"additionalProperties\" : false\n}","exceptionClassName":"io.airbyte.server.errors.BadObjectSchemaKnownException","exceptionStack":["io.airbyte.server.errors.BadObjectSchemaKnownException: The provided configuration does not fulfill the specification. Errors: json schema validation failed when comparing the data to the json schema. ","Errors: $.client_id: is missing but it is required, $.client_secret: is missing but it is required ","Schema: ","{","  \"type\" : \"object\",","  \"title\" : \"Salesforce Source Spec\",","  \"$schema\" : \"http://json-schema.org/draft-07/schema#\",","  \"required\" : [ \"client_id\", \"client_secret\", \"refresh_token\", \"start_date\", \"api_type\" ],","  \"properties\" : {","    \"api_type\" : {","      \"enum\" : [ \"BULK\", \"REST\" ],","      \"type\" : \"string\",","      \"default\" : \"BULK\",","      \"description\" : \"Unless you know that you are transferring a very small amount of data, prefer using the BULK API. This will help avoid using up all of your API call quota with Salesforce. Valid values are BULK or REST.\"","    },","    \"client_id\" : {","      \"type\" : \"string\",","      \"description\" : \"The Consumer Key that can be found when viewing your app in Salesforce\"","    },","    \"is_sandbox\" : {","      \"type\" : \"boolean\",","      \"default\" : false,","      \"description\" : \"Whether or not the the app is in a Salesforce sandbox. If you do not know what this, assume it is false. We provide more info on this field in the <a href=\\\"https://docs.airbyte.io/integrations/destinations/salesforce#is_sandbox\\\">docs</a>.\"","    },","    \"start_date\" : {","      \"type\" : \"string\",","      \"pattern\" : \"^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}Z$\",","      \"examples\" : [ \"2021-07-25T00:00:00Z\" ],","      \"description\" : \"UTC date and time in the format 2017-01-25T00:00:00Z. Any data before this date will not be replicated. This field uses the \\\"updated\\\" field if available, otherwise the \\\"created\\\" fields if they are available for a stream.\"","    },","    \"client_secret\" : {","      \"type\" : \"string\",","      \"description\" : \"The Consumer Secret that can be found when viewing your app in Salesforce\",","      \"airbyte_secret\" : true","    },","    \"refresh_token\" : {","      \"type\" : \"string\",","      \"description\" : \"Salesforce Refresh Token used for Airbyte to access your Salesforce account. If you don't know what this is, follow this <a href=\\\"https://medium.com/@bpmmendis94/obtain-access-refresh-tokens-from-salesforce-rest-api-a324fe4ccd9b\\\">guide</a> to retrieve it.\",","      \"airbyte_secret\" : true","    }","  },","  \"additionalProperties\" : false","}",
"\tat io.airbyte.cloud.auth.AuthHelper.execute(AuthHelper.java:107)",
"\tat io.airbyte.cloud.auth.AuthHelper.callForWorkspace(AuthHelper.java:78)",
"\tat io.airbyte.server.wrapped.ConfigurationApiWrapped.checkConnectionToSourceForUpdate(ConfigurationApiWrapped.java:429)",
"\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)",
"\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)",
"\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)",
"\tat java.base/java.lang.reflect.Method.invoke(Method.java:564)",
"\tat org.glassfish.jersey.server.model.internal.ResourceMethodInvocationHandlerFactory.lambda$static$0(ResourceMethodInvocationHandlerFactory.java:52)",
"\tat org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher$1.run(AbstractJavaResourceMethodDispatcher.java:124)",
"\tat org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher.invoke(AbstractJavaResourceMethodDispatcher.java:167)",
"\tat org.glassfish.jersey.server.model.internal.JavaResourceMethodDispatcherProvider$TypeOutInvoker.doDispatch(JavaResourceMethodDispatcherProvider.java:219)",
"\tat org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher.dispatch(AbstractJavaResourceMethodDispatcher.java:79)",
"\tat org.glassfish.jersey.server.model.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:469)",
"\tat org.glassfish.jersey.server.model.ResourceMethodInvoker.apply(ResourceMethodInvoker.java:391)",
"\tat org.glassfish.jersey.server.model.ResourceMethodInvoker.apply(ResourceMethodInvoker.java:80)",
"\tat org.glassfish.jersey.server.ServerRuntime$1.run(ServerRuntime.java:253)",
"\tat org.glassfish.jersey.internal.Errors$1.call(Errors.java:248)",
"\tat org.glassfish.jersey.internal.Errors$1.call(Errors.java:244)",
"\tat org.glassfish.jersey.internal.Errors.process(Errors.java:292)",
"\tat org.glassfish.jersey.internal.Errors.process(Errors.java:274)",
"\tat org.glassfish.jersey.internal.Errors.process(Errors.java:244)",
"\tat org.glassfish.jersey.process.internal.RequestScope.runInScope(RequestScope.java:265)",
"\tat org.glassfish.jersey.server.ServerRuntime.process(ServerRuntime.java:232)",
"\tat org.glassfish.jersey.server.ApplicationHandler.handle(ApplicationHandler.java:680)",
"\tat org.glassfish.jersey.servlet.WebComponent.serviceImpl(WebComponent.java:394)",
"\tat org.glassfish.jersey.servlet.WebComponent.service(WebComponent.java:346)",
"\tat org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:366)",
"\tat org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:319)",
"\tat org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:205)",
"\tat org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:763)",
"\tat org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:569)",
"\tat org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:233)",
"\tat org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1377)",
"\tat org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:188)",
"\tat org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:507)",
"\tat org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:186)",
"\tat org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1292)",
"\tat org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141)",
"\tat org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:127)",
"\tat org.eclipse.jetty.server.Server.handle(Server.java:501)",
"\tat org.eclipse.jetty.server.HttpChannel.lambda$handle$1(HttpChannel.java:383)",
"\tat org.eclipse.jetty.server.HttpChannel.dispatch(HttpChannel.java:556)",
"\tat org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:375)",
"\tat org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:273)",
"\tat org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:311)",
"\tat org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:105)",
"\tat org.eclipse.jetty.io.ChannelEndPoint$1.run(ChannelEndPoint.java:104)",
"\tat org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.runTask(EatWhatYouKill.java:336)",
"\tat org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:313)",
"\tat org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:171)",
"\tat org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:129)",
"\tat org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:375)",
"\tat org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:806)",
"\tat org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:938)",
"\tat java.base/java.lang.Thread.run(Thread.java:832)","Caused by: io.airbyte.validation.json.JsonValidationException: json schema validation failed when comparing the data to the json schema. ","Errors: $.client_id: is missing but it is required, $.client_secret: is missing but it is required ","Schema: ","{","  \"type\" : \"object\",","  \"title\" : \"Salesforce Source Spec\",","  \"$schema\" : \"http://json-schema.org/draft-07/schema#\",","  \"required\" : [ \"client_id\", \"client_secret\", \"refresh_token\", \"start_date\", \"api_type\" ],","  \"properties\" : {","    \"api_type\" : {","      \"enum\" : [ \"BULK\", \"REST\" ],","      \"type\" : \"string\",","      \"default\" : \"BULK\",","      \"description\" : \"Unless you know that you are transferring a very small amount of data, prefer using the BULK API. This will help avoid using up all of your API call quota with Salesforce. Valid values are BULK or REST.\"","    },","    \"client_id\" : {","      \"type\" : \"string\",","      \"description\" : \"The Consumer Key that can be found when viewing your app in Salesforce\"","    },","    \"is_sandbox\" : {","      \"type\" : \"boolean\",","      \"default\" : false,","      \"description\" : \"Whether or not the the app is in a Salesforce sandbox. If you do not know what this, assume it is false. We provide more info on this field in the <a href=\\\"https://docs.airbyte.io/integrations/destinations/salesforce#is_sandbox\\\">docs</a>.\"","    },","    \"start_date\" : {","      \"type\" : \"string\",","      \"pattern\" : \"^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}Z$\",","      \"examples\" : [ \"2021-07-25T00:00:00Z\" ],","      \"description\" : \"UTC date and time in the format 2017-01-25T00:00:00Z. Any data before this date will not be replicated. This field uses the \\\"updated\\\" field if available, otherwise the \\\"created\\\" fields if they are available for a stream.\"","    },","    \"client_secret\" : {","      \"type\" : \"string\",","      \"description\" : \"The Consumer Secret that can be found when viewing your app in Salesforce\",","      \"airbyte_secret\" : true","    },","    \"refresh_token\" : {","      \"type\" : \"string\",","      \"description\" : \"Salesforce Refresh Token used for Airbyte to access your Salesforce account. If you don't know what this is, follow this <a href=\\\"https://medium.com/@bpmmendis94/obtain-access-refresh-tokens-from-salesforce-rest-api-a324fe4ccd9b\\\">guide</a> to retrieve it.\",","      \"airbyte_secret\" : true","    }","  },","  \"additionalProperties\" : false","}",
"\tat io.airbyte.validation.json.JsonSchemaValidator.ensure(JsonSchemaValidator.java:68)",
"\tat io.airbyte.server.handlers.SchedulerHandler.checkSourceConnectionFromSourceIdForUpdate(SchedulerHandler.java:172)",
"\tat io.airbyte.server.wrapped.ConfigurationApiWrapped.lambda$checkConnectionToSourceForUpdate$25(ConfigurationApiWrapped.java:432)",
"\tat io.airbyte.cloud.auth.AuthHelper.execute(AuthHelper.java:101)","\t... 54 more"],"rootCauseExceptionClassName":"java.lang.Class","rootCauseExceptionStack":["io.airbyte.validation.json.JsonValidationException: json schema validation failed when comparing the data to the json schema. ","Errors: $.client_id: is missing but it is required, $.client_secret: is missing but it is required ","Schema: ","{","  \"type\" : \"object\",","  \"title\" : \"Salesforce Source Spec\",","  \"$schema\" : \"http://json-schema.org/draft-07/schema#\",","  \"required\" : [ \"client_id\", \"client_secret\", \"refresh_token\", \"start_date\", \"api_type\" ],","  \"properties\" : {","    \"api_type\" : {","      \"enum\" : [ \"BULK\", \"REST\" ],","      \"type\" : \"string\",","      \"default\" : \"BULK\",","      \"description\" : \"Unless you know that you are transferring a very small amount of data, prefer using the BULK API. This will help avoid using up all of your API call quota with Salesforce. Valid values are BULK or REST.\"","    },","    \"client_id\" : {","      \"type\" : \"string\",","      \"description\" : \"The Consumer Key that can be found when viewing your app in Salesforce\"","    },","    \"is_sandbox\" : {","      \"type\" : \"boolean\",","      \"default\" : false,","      \"description\" : \"Whether or not the the app is in a Salesforce sandbox. If you do not know what this, assume it is false. We provide more info on this field in the <a href=\\\"https://docs.airbyte.io/integrations/destinations/salesforce#is_sandbox\\\">docs</a>.\"","    },","    \"start_date\" : {","      \"type\" : \"string\",","      \"pattern\" : \"^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}Z$\",","      \"examples\" : [ \"2021-07-25T00:00:00Z\" ],","      \"description\" : \"UTC date and time in the format 2017-01-25T00:00:00Z. Any data before this date will not be replicated. This field uses the \\\"updated\\\" field if available, otherwise the \\\"created\\\" fields if they are available for a stream.\"","    },","    \"client_secret\" : {","      \"type\" : \"string\",","      \"description\" : \"The Consumer Secret that can be found when viewing your app in Salesforce\",","      \"airbyte_secret\" : true","    },","    \"refresh_token\" : {","      \"type\" : \"string\",","      \"description\" : \"Salesforce Refresh Token used for Airbyte to access your Salesforce account. If you don't know what this is, follow this <a href=\\\"https://medium.com/@bpmmendis94/obtain-access-refresh-tokens-from-salesforce-rest-api-a324fe4ccd9b\\\">guide</a> to retrieve it.\",","      \"airbyte_secret\" : true","    }","  },","  \"additionalProperties\" : false","}",
"\tat io.airbyte.validation.json.JsonSchemaValidator.ensure(JsonSchemaValidator.java:68)",
"\tat io.airbyte.server.handlers.SchedulerHandler.checkSourceConnectionFromSourceIdForUpdate(SchedulerHandler.java:172)",
"\tat io.airbyte.server.wrapped.ConfigurationApiWrapped.lambda$checkConnectionToSourceForUpdate$25(ConfigurationApiWrapped.java:432)",
"\tat io.airbyte.cloud.auth.AuthHelper.execute(AuthHelper.java:101)",
"\tat io.airbyte.cloud.auth.AuthHelper.callForWorkspace(AuthHelper.java:78)",
"\tat io.airbyte.server.wrapped.ConfigurationApiWrapped.checkConnectionToSourceForUpdate(ConfigurationApiWrapped.java:429)",
"\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)",
"\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)",
"\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)",
"\tat java.base/java.lang.reflect.Method.invoke(Method.java:564)",
"\tat org.glassfish.jersey.server.model.internal.ResourceMethodInvocationHandlerFactory.lambda$static$0(ResourceMethodInvocationHandlerFactory.java:52)",
"\tat org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher$1.run(AbstractJavaResourceMethodDispatcher.java:124)",
"\tat org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher.invoke(AbstractJavaResourceMethodDispatcher.java:167)",
"\tat org.glassfish.jersey.server.model.internal.JavaResourceMethodDispatcherProvider$TypeOutInvoker.doDispatch(JavaResourceMethodDispatcherProvider.java:219)",
"\tat org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher.dispatch(AbstractJavaResourceMethodDispatcher.java:79)",
"\tat org.glassfish.jersey.server.model.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:469)",
"\tat org.glassfish.jersey.server.model.ResourceMethodInvoker.apply(ResourceMethodInvoker.java:391)",
"\tat org.glassfish.jersey.server.model.ResourceMethodInvoker.apply(ResourceMethodInvoker.java:80)",
"\tat org.glassfish.jersey.server.ServerRuntime$1.run(ServerRuntime.java:253)",
"\tat org.glassfish.jersey.internal.Errors$1.call(Errors.java:248)",
"\tat org.glassfish.jersey.internal.Errors$1.call(Errors.java:244)",
"\tat org.glassfish.jersey.internal.Errors.process(Errors.java:292)",
"\tat org.glassfish.jersey.internal.Errors.process(Errors.java:274)",
"\tat org.glassfish.jersey.internal.Errors.process(Errors.java:244)",
"\tat org.glassfish.jersey.process.internal.RequestScope.runInScope(RequestScope.java:265)",
"\tat org.glassfish.jersey.server.ServerRuntime.process(ServerRuntime.java:232)",
"\tat org.glassfish.jersey.server.ApplicationHandler.handle(ApplicationHandler.java:680)",
"\tat org.glassfish.jersey.servlet.WebComponent.serviceImpl(WebComponent.java:394)",
"\tat org.glassfish.jersey.servlet.WebComponent.service(WebComponent.java:346)",
"\tat org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:366)",
"\tat org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:319)",
"\tat org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:205)",
"\tat org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:763)",
"\tat org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:569)",
"\tat org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:233)",
"\tat org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1377)",
"\tat org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:188)",
"\tat org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:507)",
"\tat org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:186)",
"\tat org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1292)",
"\tat org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141)",
"\tat org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:127)",
"\tat org.eclipse.jetty.server.Server.handle(Server.java:501)",
"\tat org.eclipse.jetty.server.HttpChannel.lambda$handle$1(HttpChannel.java:383)",
"\tat org.eclipse.jetty.server.HttpChannel.dispatch(HttpChannel.java:556)",
"\tat org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:375)",
"\tat org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:273)",
"\tat org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:311)",
"\tat org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:105)",
"\tat org.eclipse.jetty.io.ChannelEndPoint$1.run(ChannelEndPoint.java:104)",
"\tat org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.runTask(EatWhatYouKill.java:336)",
"\tat org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:313)",
"\tat org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:171)",
"\tat org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:129)",
"\tat org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:375)",
"\tat org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:806)",
"\tat org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:938)",
"\tat java.base/java.lang.Thread.run(Thread.java:832)"]}

```

--- a/airbyte-webapp/src/views/Connector/ServiceForm/useBuildForm.tsx
+++ b/airbyte-webapp/src/views/Connector/ServiceForm/useBuildForm.tsx
@@ -33,7 +33,8 @@ function useBuildInitialSchema(
           (schema as any).is_auth = true;
           const schemaWithoutPaths = removeNestedPaths(
             schema,
-            spec.oauthFlowInitParameters ?? []
+            spec.oauthFlowInitParameters ?? [],
+            false
           );
 
           const schemaWithoutOutputPats = removeNestedPaths(

